updating/updating_a_cluster/updating-hardware-on-nodes-running-on-vsphere.adoc

```diff
@@ -20,7 +20,7 @@ You can update your virtual hardware immediately or schedule an update in vCente
 ====
 * Version {product-version} of {product-title} requires VMware virtual hardware version 15 or later.
 
-* Before upgrading OpenShift 4.12 to OpenShift 4.13, you must update vSphere to *v7.0.2 or later*; otherwise, the OpenShift 4.12 cluster is marked *un-upgradeable*.
+* Before upgrading OpenShift 4.12 to OpenShift 4.13, you must update vSphere to *v8.0 Update 1 or later*; otherwise, the OpenShift 4.12 cluster is marked *un-upgradeable*.
```

Comment:
Is this out of date? @jiajliu could you help to take a look? Thanks

Reply:
@sgaoshang OpenShift 4.12 is currently in Extended Update Support Term 2. To update a 4.12 cluster on vSphere to 4.14, it must first be updated to 4.13(EOL) and then 4.14. More info can be checked in https://access.redhat.com/support/policy/updates/openshift#dates